core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -766,6 +771,24 @@ constructor(
         }
     }
 
+    override suspend fun setPreviewStabilization(previewStabilization: Stabilization) {
+        currentSettings.update { old ->
+            old?.copy(previewStabilization = previewStabilization)
+        }
```

Comment:
Similar to `setDynamicRange`, we should have a function that we call here called `tryApplyStabilizationConstraints` that uses `CameraConstraints.supportedStabilizationModes` to ensure we're not applying unsupported stabilization modes.